R/teal_lockfile.R

```diff
@@ -39,50 +39,23 @@ teal_lockfile <- function() {
     }
   }
 
-  if (!(is_in_test() || is_r_cmd_check())) {
```

Comment:
`is_in_test()` and `is_r_cmd_check()` no longer needed as there is no parallel cluster socket process that gets started with `future` plan